nexus_demos/config/workcell_1_bts/place_on_conveyor.xml

```diff
@@ -2,6 +2,8 @@
 <root main_tree_to_execute="PickAndPlace">
   <BehaviorTree ID="PickAndPlace">
     <Sequence name="root_sequence">
+      <WaitForSignal name="wait_for_pallet" signal="transporter_done" clear="true"/>
```

Comment:
Could you help me understand how moving the wait to the beginning of the process  resolves the issue where a different AMR reaches the workcell before the first task's AMR?

Say a workcell starts this `place_on_conveyor` process when it beings executing a `WorkcellTask` with an associated `task_id` of `task1`. To ensure the above, I would image we would need the AMR that arrives at this workcell to be executing a `WorkcellTask` with the same `task_id`?

Reply:
Per discussion, with parallel workcell tasks running on each workcell, to prevent picking all labels before any of the robots arrive, we only start the entire BT when the robot announces that it has reached the workcell

Regarding the semantics of workcell tasks (only 1 at a time, and just requires orchestrating), I've opened https://github.com/osrf/nexus/issues/78, as a continuation. We can merge this and continue to iterate towards the ideal set up.

Reply:
I thought about this more and it feels like a bit of a regression moving the wait to the top of the process as now `nexus_demos` with the `mock_transporter` will also wait for material to arrive at the workcell first. 

Could we create another copy of this BT and call it `place_on_amr.xml` with the WaitForSignal as you have it here? Then update the workcell launch to remap it's task to the `place_on_amr.xml` if `use_rmf_transporter` is true?

Reply:
Good point, changed with https://github.com/osrf/nexus/pull/73/commits/a96d43afda4875f889a0aba1c3011371e2d6543d